gammapy/irf/tests/test_psf_analytical.py

```diff
@@ -35,3 +45,25 @@ def test_EnergyDependentMultiGaussPSF_write(tmpdir):
     # see e.g. https://travis-ci.org/gammapy/gammapy/jobs/31056341#L1162
     # assert hdu_list[1].verify_checksum() == 1
     assert len(hdu_list) == 2
+
+
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+@pytest.mark.parametrize(('energy'), ENERGIES)
+def test_to_table_psf(energy):
+    filename = gammapy_extra.filename('test_datasets/unbundled/irfs/psf.fits')
+    psf = EnergyDependentMultiGaussPSF.read(filename, hdu='POINT SPREAD FUNCTION')
+    theta = Angle(0, 'deg')
+
+    table_psf = psf.to_table_psf(theta)
+    table_psf_at_energy = table_psf.table_psf_at_energy(energy)
+    psf_at_energy = psf.psf_at_energy_and_theta(energy, theta)
+    
+    # Plot containment radius vs. containment
```

Comment:
Remove this comment? (You're not plotting anything in this test, probably this is a copy & paste remnant)